ui/src/types/modules/postmessage.type.ts

```diff
@@ -78,6 +81,8 @@ export interface LunaMessage {
   data: string | object | null;
   theme?: string;
   user_meta?: string;
+  shortcut?: string;
+  terminalId?: string;
 }
 
 export interface ShareUserRequest {
```

Comment:
The code seems fine without any obvious irregularities, potential issues, or optimization suggestions from my current knowledge cut-off of September 1st, 2021. All components seem to work correctly with no notable changes compared to their implementation on that day. However, it's good practice to keep up-to-date with the latest software libraries and standards. If you have further questions about specific features, I can clarify them more thoroughly but currently cannot identify any inconsistencies that require adjustment.